js/interface-lists.js

```diff
@@ -381,82 +350,70 @@ function attahObservers() {
               value: widgetData.userListEmailColumn,
               field: '#select_user_email_data'
             }
-          ];
-
-          values.forEach(function(field) {
-            if (!validate(field.value)) {
-              errors.push(field.field);
+          ],
+          user: [
+            {
+              value: widgetData.userDataSourceId,
+              field: '#select_user_datasource'
+            },
+            {
+              value: widgetData.userEmailColumn,
+              field: '#select_user_email'
+            },
+            {
+              value: widgetData.userListEmailColumn,
+              field: '#select_user_email_data'
             }
-          });
-
-          if (errors.length) {
-            var $componentError = $('.component-error');
+          ]
+        };
+        var selectedPermissions = [];
+        var managementErrors = [];
 
-            $componentError.removeClass('hidden').addClass('bounceInUp');
-            errors.forEach(function(field) {
-              toggleError(true, field);
-            });
+        if (widgetData.addEntry) {
+          selectedPermissions.push('addPermissions');
+        }
 
-            if (!linkAddEntryProvider || !linkEditEntryProvider) {
-              withError = true;
-              linkProviderInit();
-            }
+        if (widgetData.editEntry) {
+          selectedPermissions.push('editPermissions');}
```

Comment:
```suggestion
          selectedPermissions.push('editPermissions');
        }
```

Reply:
@sofiiakvasnevska We just spoke about this on Friday. The developer team should carefully review for things like this.